src/main/groovy/nebula/plugin/dependencylock/tasks/GenerateLockTask.groovy

```diff
@@ -102,7 +102,7 @@ class GenerateLockTask extends AbstractLockTask {
                         it.canBeResolved && !ConfigurationFilters.safelyHasAResolutionAlternative(it)
                     } else {
                         //TODO: we need a better approach for global locks because `default` is going away
```

Comment:
This line can now go away. It talks of the `default` configuration, and we're now using `aggregate`